pkg/gen/input/makefile/internal/file/makefile.go

```diff
@@ -23,134 +20,5 @@ var makefileTemplate = `# DO NOT EDIT. Generated with:
-	@echo "====> $@"
-## build-linux: builds a local binary for linux/amd64
-build-linux: $(APPLICATION)-linux
-	@echo "====> $@"
-
-$(APPLICATION): $(APPLICATION)-v$(VERSION)-$(OS)-amd64
-	@echo "====> $@"
-	cp -a $< $@
-
-$(APPLICATION)-darwin: $(APPLICATION)-v$(VERSION)-darwin-amd64
-	@echo "====> $@"
-	cp -a $< $@
-
-$(APPLICATION)-linux: $(APPLICATION)-v$(VERSION)-linux-amd64
-	@echo "====> $@"
-	cp -a $< $@
-
-$(APPLICATION)-v$(VERSION)-%-amd64: $(SOURCES)
-	@echo "====> $@"
-	CGO_ENABLED=0 GOOS=$* GOARCH=amd64 go build -ldflags "$(LDFLAGS)" -o $@ .
-
-{{- if .IsFlavourCLI }}
-
-.PHONY: package-darwin package-linux
-## package-darwin: prepares a packaged darwin/amd64 version
-package-darwin: $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-darwin-amd64.tar.gz
-	@echo "====> $@"
-## package-linux: prepares a packaged linux/amd64 version
-package-linux: $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-linux-amd64.tar.gz
-	@echo "====> $@"
-
-$(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-%-amd64.tar.gz: DIR=$(PACKAGE_DIR)/$<
-$(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-%-amd64.tar.gz: $(APPLICATION)-v$(VERSION)-%-amd64
-	@echo "====> $@"
-	mkdir -p $(DIR)
-	cp $< $(DIR)/$(APPLICATION)
-	cp README.md LICENSE $(DIR)
-	tar -C $(PACKAGE_DIR) -cvzf $(PACKAGE_DIR)/$<.tar.gz $<
-	rm -rf $(DIR)
-	rm -rf $<
-
-{{- end }}
-
-.PHONY: install
-## install: install the application
-install:
-	@echo "====> $@"
-	go install -ldflags "$(LDFLAGS)" .
-
-.PHONY: run
-## run: runs go run main.go
-run:
-	@echo "====> $@"
-	go run -ldflags "$(LDFLAGS)" -race .
-
-.PHONY: clean
-## clean: cleans the binary
-clean:
-	@echo "====> $@"
-	rm -f $(APPLICATION)*
-	go clean
-
-.PHONY: imports
-## imports: runs goimports
-imports:
-	@echo "====> $@"
-	goimports -local $(shell go list .) -w .
-
-.PHONY: lint
-## lint: runs golangci-lint
-lint:
-	@echo "====> $@"
-	golangci-lint run -E gosec -E goconst --timeout=15m ./...
-
-.PHONY: test
-## test: runs go test with default values
-test:
-	@echo "====> $@"
-	go test -ldflags "$(LDFLAGS)" -race ./...
-
-.PHONY: build-docker
-## build-docker: builds docker image to registry
-build-docker: build-linux
-	@echo "====> $@"
-	docker build -t ${APPLICATION}:${VERSION} .
-
-{{- if or .IsFlavourApp }}
-
-.PHONY: lint-chart
-## lint-chart: runs ct against the default chart
-lint-chart: IMAGE := giantswarm/helm-chart-testing:v3.0.0-rc.1
-lint-chart:
-	@echo "====> $@"
-	rm -rf /tmp/$(APPLICATION)-test
-	mkdir -p /tmp/$(APPLICATION)-test/helm
-	cp -a ./helm/$(APPLICATION) /tmp/$(APPLICATION)-test/helm/
-	architect helm template --dir /tmp/$(APPLICATION)-test/helm/$(APPLICATION)
-	docker run -it --rm -v /tmp/$(APPLICATION)-test:/wd --workdir=/wd --name ct $(IMAGE) ct lint --validate-maintainers=false --charts="helm/$(APPLICATION)"
-	rm -rf /tmp/$(APPLICATION)-test
-
-{{- end }}
-
-.PHONY: help
-## help: prints this help message
-help:
-	@echo "Usage: \n"
-	@sed -n 's/^##//p' ${MAKEFILE_LIST} | column -t -s ':' |  sed -e 's/^/ /'
+include *.mk
```

Comment:
All of that is simply moved to `pkg/gen/input/makefile/internal/file/makefile_go_mk.go` and replaced with `include *.mk`.